src/core/src/UsdNotificationHandler.cpp

```diff
@@ -551,6 +585,13 @@ void processCesiumPolygonRasterOverlayRemoved(Context& context, const pxr::SdfPa
     updateRasterOverlayBindings(context, polygonRasterOverlayPath);
 }
 
+void processCesiumWebMapServiceRasterOverlayRemoved(
+    Context& context,
+    const pxr::SdfPath& webMapServiceRasterOverlayPath) {
+    context.getAssetRegistry().removeWebMapServiceRasterOverlay(webMapServiceRasterOverlayPath);
+    updateRasterOverlayBindings(context, webMapServiceRasterOverlayPath);
+}
```

Comment:
Make sure that when you merge in main that

```
    if (context.getAssetRegistry().getWebMapServiceRasterOverlay(webMapServiceRasterOverlayPath)) {
        return;
    }
```

is added

Reply:
Added. I think it goes in `processCesiumWebMapServiceRasterOverlayAdded`